### PR TITLE
Configure Global Session Policy and authentication policy guide updates

### PR DESCRIPTION
**Pull request by susanharper-okta**
## Description:
- **What's changed?** Updated guide with name changes and overview of how the GSP and auth policies work together. 
- **Is this PR related to a Monolith release?** Yes. 2022.03.0

Netlify link: https://6218056c93c2b9008b686db1--reverent-murdock-829d24.netlify.app/docs/guides/configure-signon-policy/main/

### Resolves:

* [OKTA-462458](https://oktainc.atlassian.net/browse/OKTA-462458)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/oie-intro/index.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ Some of the new capabilities unlocked by Identity Engine include:
 
 * App-level policies
 
-  An app sign-on policy is a [security policy framework](https://csrc.nist.gov/publications/detail/sp/800-63b/final) that allows organizations to model security outcomes for an app. For example, you can automatically step up authentication to a strong non-phishable factor when elevated risk is detected. See [App sign-on policies](https://help.okta.com/okta_help.htm?type=oie&id=ext-about-asop).
+  An authentication policy is a [security policy framework](https://csrc.nist.gov/publications/detail/sp/800-63b/final) that allows organizations to model security outcomes for an app. For example, you can automatically step up authentication to a strong non-phishable factor when elevated risk is detected. See [App sign-on policies](https://help.okta.com/okta_help.htm?type=oie&id=ext-about-asop).
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-intro/index.md:30]: Will Product docs still use the "App sign-on policies" title after this feature is released?

File: packages/@okta/vuepress-site/docs/reference/api/authn/index.md [hunk 2]
```diff
@@ -11,12 +11,12 @@ The Okta Authentication API provides operations to authenticate users, perform m
...
+* **Multifactor authentication** (MFA) strengthens the security of password-based authentication by requiring additional verification of another Factor such as a temporary one-time password or an SMS passcode. The Authentication API supports user enrollment with MFA factors enabled by the administrator, as well as MFA challenges based on your Global Session Policy.
 * **Recovery** allows users to securely reset their password if they've forgotten it, or unlock their account if it has been locked out due to excessive failed login attempts. This functionality is subject to the security policy set by the administrator.
 
 ## Application types
 
-The behavior of the Okta Authentication API varies depending on the type of your application and your org's security policies such as the **Okta Sign-On Policy**, **MFA Enrollment Policy**, or **Password Policy**.
+The behavior of the Okta Authentication API varies depending on the type of your application and your org's security policies such as the Global Session Policy, MFA Enrollment Policy, or Password Policy.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/reference/api/authn/index.md:19]:
```suggestion
The behavior of the Okta Authentication API varies depending on the type of your application and your org's security policies such as the Global Session Policy, the MFA Enrollment Policy, or the Password Policy.
```

File: packages/@okta/vuepress-site/docs/reference/okta-expression-language-in-identity-engine/index.md [hunk 3]
```diff
@@ -55,7 +55,7 @@ See [Integrate with Endpoint Detection and Response solutions
 
 ### Security Context
 
-You can specify certain [rule conditions](/docs/reference/api/policy/#conditions) in [app sign-on policies](/docs/reference/api/policy/#app-sign-on-policy) using expressions based on the Security Context of the app sign-on request. Security Context is made up of the [risk level](https://help.okta.com/okta_help.htm?id=csh-risk-scoring) and the matching [User behaviors](https://help.okta.com/okta_help.htm?id=ext_proc_security_behavior_detection) for the request.
+You can specify certain [rule conditions](/docs/reference/api/policy/#conditions) in [authentication policies](/docs/reference/api/policy/#authentication-policy) using expressions based on the Security Context of the app sign-on request. Security Context is made up of the [risk level](https://help.okta.com/okta_help.htm?id=csh-risk-scoring) and the matching [User behaviors](https://help.okta.com/okta_help.htm?id=ext_proc_security_behavior_detection) for the request.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/reference/okta-expression-language-in-identity-engine/index.md:58]: Is the heading "App sign-on policy" going to change to "Authentication policy" in the `/docs/reference/api/policy` page?

File: packages/@okta/vuepress-site/docs/concepts/policies/index.md [hunk 4]
```diff
@@ -21,29 +21,29 @@ There are many possibilities for policy use:
 
 Okta supports the following policy types:
 
-* [Okta Sign On Policy](/docs/reference/api/policy/#okta-sign-on-policy) &mdash; Controls who can sign in and how a user is allowed to sign in to Okta, including whether they are challenged for MFA and how long they are allowed to remain signed in before re-authenticating.
+* [Global Session Policy](/docs/reference/api/policy/#global-session-policy): Controls who can sign in and how a user is allowed to sign in to Okta, including whether they are challenged for MFA and how long they are allowed to remain signed in before re-authenticating.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/concepts/policies/index.md:24]: Is the heading "Okta sign-on policy" going to change to "Global Session policy" in the `/docs/reference/api/policy` page?

**Review by vanngo-okta — APPROVED** [review]
Great work on this topic! I learned a lot going through these edits 👍 .
A few nits and questions, but nothing big enough to hold this back: approving! 🚢

File: packages/@okta/vuepress-site/docs/guides/oie-intro/index.md (hunk 1 above)
Comment (susanharper-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-intro/index.md:30]: No, this is another one that I need to update after I get an alias from Graham.

File: packages/@okta/vuepress-site/docs/reference/okta-expression-language-in-identity-engine/index.md (hunk 3 above)
Comment (susanharper-okta) [reply packages/@okta/vuepress-site/docs/reference/okta-expression-language-in-identity-engine/index.md:58]: Yes, that's part of the other branch I have going with the Dev team.

**Review by susanharper-okta — COMMENTED** [review]
responses

File: packages/@okta/vuepress-site/docs/guides/configure-signon-policy/main/index.md [hunk 5]
```diff
@@ -1,152 +1,151 @@
...
+6. Configure THEN conditions to define the authentication experience. For this use case, leave the default of **Allowed after successful authentication** for **THEN Access is**.
 
-11. Click **Save**.
+7. In the **Re-authentication frequency** section, select **Every sign-in attempt** for both **AND Password re-authentication frequency is** and **AND Re-authentication frequency for all other factors is**.
 
-## Next steps
+8. Click **Save**.
 
-You should now understand how to add an Okta Sign-On Policy and an App Sign-On Policy.
+> **Note:** You can [use the API](/docs/reference/api/policy/#app-sign-on-policy) to assign an app to an authentication policy. You need the application ID and the policy ID for this API request. Make a `PUT /api/v1/apps/${appId}/policies/${policyId}` request. No HTTP body is necessary for the PUT request. Then, to check that the assignment was successful, make a `GET /api/v1/apps/${appId}` request and the response should contain information on the policy associated with the app.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/configure-signon-policy/main/index.md:145]:
```suggestion
> **Note:** You can [use the API](/docs/reference/api/policy/#authentication-policy) to assign an app to an authentication policy. You need the application ID and the policy ID for this API request. Make a `PUT /api/v1/apps/${appId}/policies/${policyId}` request. No HTTP body is necessary for the PUT request. Then, to check that the assignment was successful, make a `GET /api/v1/apps/${appId}` request and the response should contain information on the policy associated with the app.
```

**Pull request merged**